commands/command_edit.cpp

```diff
@@ -112,7 +112,11 @@ void EditBase<T>::undo()
 
 	// Send signals.
 	DiveField id = fieldId();
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+	emit diveListNotifier.divesChanged(QVector<dive *>(dives.begin(), dives.end()), id);
```

Comment:
What? Why is the old method deprecated? Isn't the old one much more readable? Sigh.